launch_testing/test/launch_testing/examples/README.md

```diff
@@ -1,12 +1,12 @@
 # Examples
 
-## `good_proc.test.py`
```

Comment:
I understand why moving the examples, but I think that the README should stay at the top level directory of the package (in `launch_testing`). It'll be more visible there.

Reply:
Sure, we can move it there. See 8014b4c94a5cded069bca06d4f031398f6c76a4b.